packages/demux/src/AloxideActionHandler.ts

```diff
@@ -30,7 +30,7 @@ export class AloxideActionHandler extends AbstractActionHandler {
 
   constructor(
     protected bcName: string,
-    protected dataAdapter: DataAdapter<any, any>,
+    protected dataAdapter: AloxideDataManager,
```

Comment:
I saw that there are two more functions that you added to `AloxideDataManager`. Why don't you just add them to `DataAdapter` and remove this change?

Reply:
With `DataAdapter` we don't know which entityNames to revert.